components/wb-fieldflow/fieldflow.js

```diff
@@ -471,8 +472,7 @@ var componentName = "wb-fieldflow",
 				label: prefLabel,
 				defaultselectedlabel: defaultSelectedLabel,
 				lblselector: lblselector,
-				items: itemsToCreate,
-				inline: data.inline
```

Comment:
Is removing the inline option for data table filtering is intentional?

Do not you want to add the gcChkboxrdio instead?

Reply:
Oops, yup that's a mistake. C'est un résultat de mon testing lol.